images/pytorch-pods/launch_k8s_workers.py

```diff
@@ -148,6 +152,15 @@ def main(argv):
     'TPU_NUM_DEVICES': '8',
   }
 
+  if FLAGS.volumes:
+    volumes = {
+      name: mount_path for name, mount_path in
+      [v.split(':') for v in FLAGS.volumes]
+    }
+    print(volumes)
+  else:
+    volumes = []
```

Comment:
if populated, it's a dict but if empty, it's a list. Probably fine since `if not volumes` will work the same but there might be some edge cases that lead to weird behavior

Reply:
Fixed.